jgrapht-core/src/test/java/org/jgrapht/GraphMetricsTest.java

```diff
@@ -234,4 +241,68 @@ public void testDirectedGraphs()
             assertEquals(minCycle, GraphMetrics.getGirth(graph));
         }
     }
+
```

Comment:
It would be good to add a few test methods where you invoke testCountTriangles on fixed input graphs for which the number of triangles is known, e.g.: cycle graph, complete graph, and some other named graphs, etc

Reply:
Done. By cycle graph do you mean wheel graph?